tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/BUILD

```diff
@@ -4,19 +4,12 @@ package(default_visibility = ["//tensorboard:internal"])
 
 licenses(["notice"])
 
-py_binary(
-    name = "inline_images",
-    srcs = ["inline_images.py"],
-    python_version = "PY3",
-    srcs_version = "PY3",
-)
-
 genrule(
     name = "gen_inactive_component.ng.html",
     srcs = ["inactive_component.uninlined.ng.html"] + glob(["images/*.png"]),
     outs = ["inactive_component.ng.html"],
-    cmd = "$(execpath //tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive:inline_images) $(SRCS) >'$@'",
-    exec_tools = [":inline_images"],
+    cmd = "$(execpath //tensorboard/defs:inline_images) $(SRCS) >'$@'",
+    exec_tools = ["//tensorboard/defs:inline_images"],
 )
```

Comment:
Great but we don't want all consumers of the tool to use genrule and this boilerplatey syntax or API of the tool.

Instead, make a macro that goes something like:
```py
# In appropriate .bzl file
def tf_inline_images(name, html_template, images, out):
    native.genrule(
        name=name,
        srcs=[html_template] + images,
        outs=[out],
        cmd="$(execpath //tensorboard/defs:inline_images) $(SRCS) >'$@'",
        exec_tools=["//tensorboard/defs:inline_images"],
    ) 
```

which you will use it like:
```py
load(<path to the bzl>, "tf_inline_images")
...


tf_inline_images(
    name="gen_inactive_component.ng.html",
    html_template = "inactive_component.uninlined.ng.html",
    images = glob(["images/*.png"]),
    out = "inactive_component.ng.html",
)
```

Reply:
Added a macro in to `defs.bzl`. Thanks for the informative comment!